### ISSUE TITLE
[Table]: focus is missing in two situations

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents-react#7006]
### Describe the bug

Two issues here:
1. focus style is missing after focusing on collapse button
2. Focus outline is not visible for the entire row in HCB mode, focus outline on the link is visible.

### Isolated Example

https://stackblitz.com/edit/github-9fhgrq5o-7ejgea6v?file=package.json

### Reproduction steps

use tab to check focus
set HCB mode and use tab to check focus

### Expected Behaviour

focus style is clear

### UI5 Web Components for React Version

2.7

### UI5 Web Components Version

2.7

### Browser

Edge

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @wilson-xu-sap

the `ObjectPage` only allows `ObjectPageSection` as child components.

> __Note:__ Although this prop accepts all HTML Elements, it is strongly recommended that you only use `ObjectPageSection` in order to preserve the intended design.

Without the `ObjectPage` I wasn't able to reproduce the reported issues. Could you please check again?
https://stackblitz.com/edit/github-9fhgrq5o-fsgcpert?file=src%2FApp.tsx,index.html

**Comment by wilson-xu-sap** [comment]
Hi @Lukas742 Thanks for explaining. I have tested issue 1 in normal mode and it works well. Could you tell me how to set HCB mode in stackblitz?
